tendrl/ceph_integration/manager/manager.py

```diff
@@ -204,7 +157,7 @@ def main():
                 cluster_id = sys.argv[2]
                 utils.set_tendrl_context(cluster_id)
```

Comment:
I am not sure at the moment and as we discussed earlier ideally we should be starting bridges as well as services and not child processes of the node agent. We can do this as part of a separate PR preferably as it applies to both gluster and ceph integration modules.